test/make.sh

```diff
@@ -71,14 +71,6 @@ function basefiles() {
   done
 }
 
-# This function runs the hadolint linter on
-# every file named 'Dockerfile'
-function docker() {
-  echo "Running hadolint on Dockerfiles"
-  find_files . -name "Dockerfile" -print0 \
-    | compat_xargs -0 hadolint
-}
-
 # This function runs 'terraform validate' and 'terraform fmt'
 # against all directory paths which contain *.tf files.
 function check_terraform() {
```

Comment:
Please change terraform fmt validation to `find_files . -name "*.tf" -exec terraform fmt  -check=true -write=false {} \;` to check `*.tf ` files, not directories, contains them because symlinks to `test/fixtures/shared/terraform.tfvars` is broken.